app/extras/queries/users_to_mention_query.rb

```diff
@@ -1,6 +1,6 @@
 class Queries::UsersToMentionQuery
   def self.for(model)
-    model.mentioned_group_members
+    model.mentioned_users
          .where.not(id: model.group.members.mentioned_in(model)) # avoid re-mentioning users when editing
```

Comment:
Have to be careful here, I think this wants to be `model.members`, so that you can't remention people in guest groups as well. Also have to make sure that stance, comment, etc. respond correctly to 'members'